### PR TITLE
basic bot that implements strict turn taking

### PR DESCRIPTION
**Pull request by sebag90**
This is an example of a simple 2 users bot that enforces strict turn taking during the conversation.
After the roles are randomly assigned at the beginning, the bot ensures that after each message the roles are switched

### CODE REVIEW

**Comment by sebag90** [comment]
TODO: modify readme

**Comment by sebag90** [comment]
the input field of the user who cannot send messages is now read-only

**Comment by sebag90** [comment]
https://github.com/clp-research/slurk/pull/269 this must be merged for this bot to work

File: strict_turn_taking/__main__.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                    "attribute": "placeholder",
+                    "value": "Enter your message here!",
+                    "receiver_id": other_usr["id"],
+                },
+                headers={"Authorization": f"Bearer {self.token}"},
+            )
+
+
+if __name__ == "__main__":
+    # set up loggingging configuration
```
Comment (janagoetze) [inline strict_turn_taking/__main__.py:224]: loggingging

Comment (sebag90) [reply strict_turn_taking/__main__.py]: fixed :+1:

**Pull request merged**